p2p/bucket/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "cuprate-p2p-bucket"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
This will need license etc, can you copy from another crate and edit appropriately